libpod/shutdown/handler.go

```diff
@@ -75,6 +83,7 @@ func Start() error {
 			}
 			handlerLock.Unlock()
 			shutdownInhibit.Unlock()
+			os.Exit(exitCode)
```

Comment:
Wouldn't this prevent a lot of `defer` functions from running? I know I avoided it deliberately when I wrote this

Reply:
I mean yes but if we exit what do we want to defer here? And most importantly the current handler also just exited so there should not be any functional difference I would say

Reply:
IIRC we have a bunch of things running in defer that do things like removing files to clean up after ourselves

Reply:
Sure but this PR doesn't change the behavior here. Critical sections where we cannot leak need to use `shutdown.Inhibit()` which is how I fixed the issue with the netns leak.